source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -113,6 +113,55 @@ Name           | Type   | Value | Description
 
 <%= partial 'localizable/smart_cart/order_object' %>
 
+## Test requests (sandbox)
+
+For [demo orders](/smart_cart/orders_api/#test-requests-sandbox), separate endpoints are available to trigger different
+kinds of webhook requests for testing. In the following examples, `:code` should be replaced with one of the following values:
+
+- DEMO-OPEN
+- DEMO-ACCEPTED
+- DEMO-REJECTED
+- DEMO-INVOICE
+- DEMO-INVOICE39A
+
+An order creation or order update event should be received at the webhook url of the shop, depending on the request action.
+
+#### New order
+
+This sends a Webhook request payload just like if the order is new, with event_type:
+“new_order”.
+
+<pre class="terminal">
```

Comment:
All of those would result to `{ "success": true }` IIRC. 
Maybe treat them like the other API requests with partials?

Reply:
That's true, maybe add a note for all of them?

Reply:
Sure!